Example/Podfile

```diff
@@ -5,13 +5,13 @@ platform :ios, '9.0'
 target 'ChatExample' do
   use_frameworks!
   pod 'MessageKit', :path => '../'
+  pod 'SwiftLint', '~> 0.39.1'
```

Comment:
Could be added also via SPM maybe?

Reply:
For reference, these are all the dependencies for `SwiftLint` added via SPM
<img width="213" alt="Screenshot 2020-03-10 at 09 11 23" src="https://user-images.githubusercontent.com/5503097/76292406-7205c100-62af-11ea-9710-c26ed9fc805a.png">